salt/metalk8s/orchestrate/deploy_node.sls

```diff
@@ -116,6 +116,13 @@ Run the highstate:
     - tgt: {{ node_name }}
     - highstate: True
     - saltenv: metalk8s-{{ version }}
+    - pillar:
+        # Add ability to override node roles to not apply all the highstate
+        # e.g.: Skipping etcd when downgrading
```

Comment:
It works, sure, but doesn't look super good IMHO... Could we use something dedicated to "skipping" roles? E.g. `pillar.highstate.skip_roles", and let the `top.sls` filter this out?

Reply:
Sure could be better i will

Reply:
Ok just thinking about it and that mean that we will not support it for current released version as we can't change the top.sls of already released version but I think it's quite ok